bin/appliance_console

```diff
@@ -117,6 +118,7 @@ module ApplianceConsole
         summary_entry("MAC Address", mac),
         summary_entry("Timezone", timezone),
         summary_entry("Local Database Server", PostgresAdmin.local_server_status),
+        summary_entry("Messaging", messaging ? "configured" : "not configured"),
```

Comment:
This ends up in the middle of the database information (the next 3 lines are DB related), so I think we should move it after that (before `"External Auth"`)